src/components/Calculator.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  return (
```

Comment:
- You have done great so far :+1: but there are a few changes you need to make before moving to the next milestone. :+1: :100: 

> Create at least 1 child component
> Calculator.js should have at least 1 child component

- [ ] You did great creating the `Calculator.js` component and styling it, but as stated in the project requirements,  the Calculator.js component needs to have at least one child component, You can extract the button or even the display input into its own component and then render it inside the `Calculator.js` where it should be. :+1: :wave: I'll suggest you look into this. :+1: :+1: